original/v1/s006-lists/lists.html

```diff
@@ -369,9 +369,13 @@ <h3 class="text-muted" id="show-all">PyLadies Brno</h3>
                             ale nebude to ten samý seznam:
                             měnit se bude nezávisle na tom starém.
                         </div>
+                        <div>
+                            Jinou možností se stejným výsledkem je použití
+                            metody <code>copy</code>.
+                        </div>
```

Comment:
V kurzu bohužel není možné obsáhnout všechno. Když jsem vybíral co na této lekci zmíním, `copy` jsem sem sem nedával záměrně. `list()` stačí, a navíc funguje na čemkoli iterovatelném a výsledek je vždycky seznam, což většinou chceš. Přijde že zrovna přidání odstavce o alternativě lidi jen zbytečně zmate.
Nebo je pro to jiný důvod, který nevidím?